### ISSUE TITLE
The inception score on coco dataset

### ISSUE THREAD

**Issue by liushanyuan18** [taoxugit/AttnGAN#46]
Hi,in my experiment,the inception score of the pretrained model on coco dataset is 16.16.Do you know why the IS is not stable?How can make the IS to 25.89?

**Comment by mkumar10** [comment]
I think your evaluation is flawed. I had that issue. Make sure your images are in RGB format and not BGR if you use opencv

**Comment by liushanyuan18** [comment]
> I think your evaluation is flawed. I had that issue. Make sure your images are in RGB format and not BGR if you use opencv

Thank you so much!You are right.
By the way,I have a question in CA(Conditioning Augmentation).We convert embedding ϕt to ĉ0,and use the KL loss between ĉ0 and N~(0,1) to make ĉ0 close to N~(0,1).Why the KL loss is necessary?If ĉ0 is far away from N~(0,1),I think this is OK and has no bad,but when I abandon KL loss,the IS and FID is not good as early.Do you know why?Thanks!

**Comment by ShihuaHuang95** [comment]
@liushanyuan18 @mkumar10 How about your scores? In my experiment, the IS of the pre-trained model on coco dataset is 23.51, it seems far from the 25.89.

**Comment by liushanyuan18** [comment]
> @liushanyuan18 @mkumar10 How about your scores? In my experiment, the IS of the pre-trained model on coco dataset is 23.51, it seems far from the 25.89.

We get 23.74 on coco dataset.It is similar to your score.
